src/plugins/data/public/ui/dataset_selector/configurator.tsx

```diff
@@ -40,10 +41,21 @@ export const Configurator = ({
   const queryString = queryService.queryString;
   const languageService = queryService.queryString.getLanguageService();
   const indexPatternsService = getIndexPatterns();
-  const type = queryString.getDatasetService().getType(baseDataset.type);
+  const type = queryString?.getDatasetService().getType(baseDataset.type);
```

Comment:
queryString manager shouldn't be optional how come? the type might be undefined but the app won't work if queryString is undefined